browser-tests/eventGroupsFeature.ts

```diff
@@ -0,0 +1,32 @@
+import { godchildrenProfilePage, route } from './pages/godchildrenProfilePage';
```

Comment:
When I was working on the project we used "Culture Kids" instead of godchildren for the English translation. You might avoid some confusion if you stay consistent with that translation everywhere in the code.